11/umbraco-cms/reference/security/backofficeusermanager-and-notifications.md

```diff
@@ -76,8 +76,8 @@ Note the constructor minimum needs to inject what is required for the base `Back
 
 ## Notifications
 
-There are [several notifications](https://apidocs.umbraco.com/v9/csharp/api/Umbraco.Cms.Web.Common.Security.BackOfficeUserManager.html) you can handle on the `BackOfficeUserManager`.
-Internally these are mostly used for auditing but there are some that allow you to customize some workflows:
+There are [many notifications](https://apidocs.umbraco.com/v11/csharp/api/Umbraco.Cms.Web.Common.Security.BackOfficeUserManager.html) you can handle on the `BackOfficeUserManager`.
```

Comment:
this change will need to be made in v10 docs as well to keep the content consistent on all versions 🙏

Reply:
I believe it's the same on all 3 major versions now.